requirements.txt

```diff
@@ -1,62 +1,56 @@
 Django==2.1.8
-Pillow==4.3.0                   # PIL fork (Python Imaging Library)
-djangorestframework==3.7.7      # API v1
+Pillow==6.0.0                   # PIL fork (Python Imaging Library)
```

Comment:
We are upgrading two major versions here, any idea what has actually been deprecated?
I found little to no information in their releases on Github.

Reply:
As far as I can see (or the IDEA can search) we only use PIL in https://github.com/dotkom/onlineweb4/blob/358b15f1bc3ec25c8dd530b9186c6ab3a169beae/apps/gallery/util.py#L10, and we use relatively few functions, that I do not think are affected by their deprecations https://pillow.readthedocs.io/en/stable/releasenotes/index.html